tests/minimal-vmtest.nix

```diff
@@ -23,9 +17,13 @@ pkgs.nixosTest {
         "${pkgs.path}/nixos/tests/common/x11.nix"
       ];
 
+      systemd.tmpfiles.rules = [
+        "f /etc/authentik.env 0700 root root - AUTHENTIK_SECRET_KEY=thissecretwillnotbeinthenixstore"
+      ];
+
       services.authentik = {
         enable = true;
-        environmentFile = authentik-env;
+        environmentFile = "/etc/authentik.env";
```

Comment:
I see that the tmpfiles rule is just there to place the file at the given location, i.e. /etc/authentik.env, like sops-nix or agenix would. But someone with not too much experience, just looking through the tests for a way to get the secret file to the correct place, might be thrown off by this and think they can also use tmpfiles to get the job done.
 
I suggest changing the secret value to an obvious placeholder and adding a short comment above the tmpfiles rule, indicating that this shouldn't be taken to production, because it serves as a replacement for using something like sops-nix/agenix or even manually deploying secrets to the desired location.

Reply:
Done.